src/main/java/com/adobe/cq/commerce/graphql/client/impl/GraphqlClientConfiguration.java

```diff
@@ -61,4 +65,22 @@
         description = "The maximum number of concurrent HTTP connections the connector can make",
         type = AttributeType.INTEGER)
     int maxHttpConnections() default MAX_HTTP_CONNECTIONS_DEFAULT;
+
+    @AttributeDefinition(
+        name = "Http connection timeout",
+        description = "The timeout in milliseconds until a connection is established. A timeout value of zero is interpreted as an infinite timeout.",
+        type = AttributeType.INTEGER)
+    int connTimeout() default DEFAULT_CONN_TIMEOUT;
```

Comment:
I'm being picky here, but better use full names, they will be easier to read for anyone new wondering in a few months what "conn" might mean: `connectionTimeout`. Please also apply the same changes for the variables in the `GraphqlClientImpl` class, thx.